psm/src/Plutus/Model/Validator/V2.hs

```diff
@@ -49,6 +61,14 @@ mkTypedPolicyPlutarch conf term = TypedPolicy . toV2 <$> mkMintingPolicyScriptPl
 mkTypedStakePlutarch :: Config -> ClosedTerm PStakeValidator -> Either Text (TypedStake redeemer)
 mkTypedStakePlutarch conf term = TypedStake . toV2 <$> mkStakeValidatorScriptPlutarch conf term
 
+mkTypedValidatorPlutarchTypedScript :: PlutarchTypedScript PValidator -> TypedValidator datum redeemer
+mkTypedValidatorPlutarchTypedScript script =
+  TypedValidator . toV2 . Validator Plutus.$ getPlutarchTypedScript script
+
+mkTypedPolicyPlutarchTypedScript :: PlutarchTypedScript PMintingPolicy -> TypedPolicy redeemer
+mkTypedPolicyPlutarchTypedScript script =
+  TypedPolicy . toV2 . MintingPolicy Plutus.$ getPlutarchTypedScript script
```

Comment:
How would you use this? That being said, having a unit test would be great!

Reply:
Unit test added!